src/main/java/uk/gov/hmcts/cmc/claimstore/services/DefaultJudgmentService.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.cmc.claimstore.events.EventProducer;
+import uk.gov.hmcts.cmc.claimstore.exceptions.ForbiddenActionException;
+import uk.gov.hmcts.cmc.claimstore.exceptions.NotFoundException;
+import uk.gov.hmcts.cmc.claimstore.models.Claim;
+import uk.gov.hmcts.cmc.claimstore.models.DefaultJudgment;
+import uk.gov.hmcts.cmc.claimstore.processors.JsonMapper;
+import uk.gov.hmcts.cmc.claimstore.repositories.DefaultJudgmentRepository;
+
+import java.time.LocalDate;
+import java.util.Map;
+
+@Component
+public class DefaultJudgmentService {
+
+    private final DefaultJudgmentRepository defaultJudgmentRepository;
+    private final ClaimService claimService;
+    private final JsonMapper jsonMapper;
+    private final EventProducer eventProducer;
+
+    @Autowired
+    public DefaultJudgmentService(
+        ClaimService claimService,
+        DefaultJudgmentRepository defaultJudgmentRepository,
+        JsonMapper jsonMapper,
+        EventProducer eventProducer) {
+        this.defaultJudgmentRepository = defaultJudgmentRepository;
+        this.claimService = claimService;
+        this.jsonMapper = jsonMapper;
+        this.eventProducer = eventProducer;
+    }
+
+    public DefaultJudgment getByClaimId(final long claimId) {
+        return defaultJudgmentRepository
+            .getByClaimId(claimId)
+            .orElseThrow(() -> new NotFoundException("Default judgment for claim " + claimId + " not found"));
+    }
+
+    @Transactional
+    public DefaultJudgment save(final long submitterId, final Map<String, Object> data, final long claimId) {
+
+        Claim claim = claimService.getClaimById(claimId);
+
+        if (!isClaimSubmittedByUser(claim, submitterId)) {
```

Comment:
this should be refactored in a follow up PR, the validation of if its allowed to proceed